genesyscloud/resource_genesyscloud_routing_queue_test.go

```diff
@@ -1607,3 +1639,30 @@ func TestAccResourceRoutingQueueSkillGroups(t *testing.T) {
 		CheckDestroy: testVerifyQueuesDestroyed,
 	})
 }
+
+func cleanupGroup(idPrefix string) {
+	GroupsApi := platformclientv2.NewGroupsApi()
+
```

Comment:
Same comments here as the other cleanup funcs. Also, the routing queue resource has been moved to its own package, so this work will have to be moved to that new folder after you merge dev into this branch